set_user--1.0.sql

```diff
@@ -23,6 +23,17 @@ RETURNS text
 AS 'MODULE_PATHNAME', 'set_user'
 LANGUAGE C STRICT;
 
+CREATE FUNCTION set_user_u(text)
```

Comment:
Since we are changing the SQL visible functions I believe we need to bump the extension version and provide an upgrade script.